pkg/cri/sbserver/sandbox_portforward_windows.go

```diff
@@ -75,3 +87,70 @@ func (c *criService) execInSandbox(ctx context.Context, sandboxID string, cmd []
 		Code: int(*exitCode),
 	}
 }
+
+func hpcPortForwarding(ctx context.Context, id string, port int32, stream io.ReadWriter) error {
+	// Dial to localhost for HPCs since host process containers
+	// share network namespace of the host
+	podIP := "127.0.0.1"
```

Comment:
Knowing nothing about how this works in Windows, does this also need to support ipv6 or just ipv4? Or does Windows have some special magic here

Reply:
even if its ipv6, it should be able to support ipv4 loopback address, right?

Reply:
For ipv6, the equivalent to 127.0.0.1 is ::1 

For now I would think this just works for ipv4 but that would need to be verified.

Reply:
not a networking expect but would replacing with "localhost" take care of the ipv4 vs ipv6 implicitly?

Reply:
if not, we should check for ipv4 vs ipv6 (not sure how this is done but will check
)

Reply:
Addressed this after testing newly pushed changes.